tiller_python_service/git_actions.py

```diff
@@ -202,6 +202,9 @@ def merge_pull_request(pull_request_url, commit_title, commit_message, merge_met
     else:
         print(f'Failed to merge pull request. Response: {response.content}')
```

Comment:
We shouldn't be just using raw prints in this file, can we replace them with calls to the `logging` library instead? help tiller